packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -60,11 +62,32 @@ export const appMusicPlaylistsModel = createModel(
                 state,
             }),
             JOIN_ROOM: (args: { roomID: string }) => args,
+            LEAVE_ROOM: (args: { roomID: string }) => args,
+            RECEIVED_FORCED_DISCONNECTION: (args: {
+                roomSummary: MpeRoomSummary;
+            }) => args,
+            RECEIVED_MPE_LEAVE_ROOM_CALLBACK: (args: {
+                roomSummary: MpeRoomSummary;
+            }) => args,
+            SPAWN_NEW_PLAYLIST_ACTOR_FROM_STATE: (args: {
+                state: MpeWorkflowState;
+            }) => args,
```

Comment:
This event can be safely deleted as it has been replaced by `ROOM_CREATION_ACKNOWLEDGEMENT` in https://github.com/AdonisEnProvence/MusicRoom/pull/359

Reply:
Thanks, merge conflict mistake I guess